### ISSUE TITLE
Importer behaviour on pressing cancel

### ISSUE THREAD

**Issue by Uafinder** [Kademi/kademi-dev#4745]
For example, I have a large file with numerous rows in it. It contains users.
I've begin to upload it, but then I think that it fair enough, and pressing cancel. It write to me, that some number of rows are processed and successfully imported, but physically they are not.
So, how it looks right? Either it shouldn't show successfully uploaded rows or it should import them.
p.s. I've done re-index, but it doesn't helps.
![image](https://user-images.githubusercontent.com/1945230/34664246-97b238dc-f463-11e7-8681-7be676351bea.png)
![image](https://user-images.githubusercontent.com/1945230/34664261-b139ebce-f463-11e7-8a4d-07deed77a380.png)
![image](https://user-images.githubusercontent.com/1945230/34664266-bcc5b19e-f463-11e7-9409-eb98a0f4d2bf.png)
![image](https://user-images.githubusercontent.com/1945230/34664272-c42406c0-f463-11e7-99bc-71d6698a28a5.png)
http://vladtest19.admin.kademi-ci.co/manageUsers/

**Comment by bradmac** [comment]
Correct, if the import is cancelled and no rows are physically imported, the summary page show have a clear message and now show imported rows
